stable/spark/templates/spark-master-configmap.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
 data:
   health_check.sh: |
 {{ include "v3io-configs.script.httpHealthCheckWithJava" . | indent 4 }}
+    # Test that dayman is alive and accessible
+    hadoop fs -test -e /
```

Comment:
Why would spark need it? Isn't it constantly connected to the daemon?

Reply:
worker connects only when job is running. the idea is to prevent failure during the first attempt to run a job after dayman restart for some reason

Reply:
I agree that for master it could be less relevant